vtex-apps/store-components
# Show savings as percentage prop added

#### What problem is this solving?
Show savings as percentage prop added

Show savings as percentage prop added
#### How should this be manually tested?

1. Make `savingsAsPercentage` prop `true` on the `product-price` block
2. Check the product page to se if it renders as a percentage discount

[Workspace](url)
https://dev--miriadeit.myvtex.com/prodotto-4/p
#### Checklist/Reminders


#### Screenshots or example usage

#### Type of changes


✔️ | Type of Change
---|---
_ | Bug fix 
✔️| New feature 
_ | Breaking change 
_ | Technical improvements 

#### Notes
